modules/auxiliary/gather/windows_secrets_dump.rb

```diff
@@ -1113,55 +1132,69 @@ def run
     check_lm_hash_not_stored if @winreg
 
     if ['ALL', 'SAM'].include?(action.name)
-      begin
-        sam = save_sam
-      rescue RubySMB::Error::RubySMBError => e
-        if action.name == 'ALL'
-          print_warning("Error when getting SAM hive... skipping ([#{e.class}] #{e}).")
+      if @winreg
+        if datastore['INLINE']
+          print_status('Using `INLINE` technique for SAM. You might want to try without if it fails.')
```

Comment:
>  You might want to try without if it fails.

This doesn't instill the sense of confidence I'd like our users to have in our product. Would it be possible to remove this from being printed whenever the technique is used and instead only display the recommendation when an error is detected making it necessary? Do you know of cases where using this technique would be the reason the action fails but by setting INLINE it would succeed?

There's a second instance of this as well.

Reply:
I agree. I'll change these. Thanks!